lib/src/validator/size.dart

```diff
@@ -8,6 +8,9 @@ import 'dart:math' as math;
 import '../entrypoint.dart';
 import '../validator.dart';
 
+import 'package:path/path.dart';
```

Comment:
Style note: the `path` package is generally imported with the prefix `p`, since it exposes a bunch of very terse names that would otherwise be likely to conflict.

Reply:
`path` will not be used in next revision. Thank you for a tip.